cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -41,6 +38,7 @@
   public static final String METRIC_REPORTER_SAMPLER_BOOTSTRAP_SERVERS = "metric.reporter.sampler.bootstrap.servers";
   public static final String METRIC_REPORTER_TOPIC_PATTERN = "metric.reporter.topic.pattern";
```

Comment:
since we change to assign from subscribe,  I feel this config is better rename to `METRIC_REPORTER_TOPIC`.

Reply:
Added `METRIC_REPORTER_TOPIC`, but also preserved `METRIC_REPORTER_TOPIC_PATTERN` for backwards compatibility -- we may want to remove the config in a future version.